scania-digital-design-system/sdds
# feat(header): search-bar

**Describe pull-request**  
Makes the searchbar collapse in to the side menu on lower breakpoints

**Solving issue**  
Fixes: [AB#1733](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1733)

**How to test**  
Check the storybook link (Header > Searchbar menu). go to a breakpoint below the threshold and test around.
